index.rst

```diff
@@ -4,22 +4,215 @@
+In this case, there are chapters (seen on the left) showing the analysis from different runs and the results that were obtained.
+This is an appropriate use of a technote.
+
+Technotes should not be used in place of user-facing how-to documentation and/or procedures. 
+This is better addressed as part of the `Obs-Ops Webspace`_ or even an addition to the collection of `Collaborative Notebooks`_.
+
+Creation and Review of Technotes
+--------------------------------
+
+When creating a technote, a Jira ticket should be created following the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_, this is primarily to facilitate an easier review process and to track progress.
+Ideally, reviews of technotes should be performed by the intended customer of the content.
+Collaborators on the work are also useful reviewers.
+
+Technote are created by private messaging the ``sqrbot-jr`` bot on Slack and sending a ``create project`` message.
+At this time, options will pop up and users can select the template they desire (AASTEX LaTeX, lsstdoc LaTeX or reStructuredText).
+The choice is up to the user, however, reStructuredText will render as a webpage at ``sitcomtn-###.lsst.io``, whereas the others will render as PDF at the same URL.
+Depending on how the document will be used and/or interacted with, there may be a preference for a specific format.
 
 .. note::
 
-   **This technote is a work-in-progress.**
+   Adding a ``\v`` to the url will display the rendered versions of all branches in the repo (e.g. ``sitcomtn-###.lsst.io/v``).
 
-Abstract
-========
+When applicable, technotes should also link and/or include all applicable supporting documentation.
+This includes any notebooks and/or the tickets where work was performed, especially if there exists content in the ticket which may be applicable to the content.
+Whether or not this is necessary should be addressed on a case-by-case basis.
+Inside the technote repository is a ``_static`` directory where other files can be included, such as any analysis Jupyter Notebook referenced by the text, or supporting figures.
+
+Technotes should *never be deleted*.
+In the event that the content becomes stale, superseded, or even found to be incorrect, a new version of the technote should be committed that removes the content and has ``DEPRECATED`` in the title.
+If applicable, the replacement document(s) should be linked.
+This way the content is never entirely lost as it will remain in the git history of the repository.
+
+The Data Management team uses technotes extensively, more information on their using of the tool(s) are found at `their dedicated webpage <https://developer.lsst.io/project-docs/technotes.html>`_.
+
+
+Jupyter Notebooks
+^^^^^^^^^^^^^^^^^
+
+Jupyter notebooks (henceforth referred to as notebooks) are used routinely during commissioning exercises for both analysis and even for certain early observatory control sequences.
```

Comment:
Maybe play the "(henceforth referred to as notebooks)" card earlier?

Reply:
Doesn't quite work... but I did add a "Jupyter" further up to relieve the possibility of confusion.